cmd/nomos/util/write.go

```diff
@@ -31,6 +31,8 @@ const (
 	NotConfiguredMsg = "NOT CONFIGURED"
 	// UnknownMsg indicates that a field's value is unknown or unavailable.
 	UnknownMsg = "UNKNOWN"
+	// OSSMsg indicates OSS installation
+	OSSMsg = "OSS Installation"
```

Comment:
I'm not sure that `OSS Installation` is the right language here. I think something along the lines of "standalone installation` is more accurate. The important distinction IMO is whether the ACM operator is installed

Reply:
Oh is this a cherry pick? I didn't realize this was already submitted. In that case don't change this PR

Reply:
I can create a separate PR to address this

Reply:
Good point. @victorpras Can you make a separate PR to change it to `Standalone Config Sync`?

Reply:
PR to change the output https://github.com/GoogleContainerTools/kpt-config-sync/pull/436